server/lib/tuist_web/live/qa_run_live.ex

```diff
@@ -307,7 +307,7 @@ defmodule TuistWeb.QARunLive do
       time_seconds =
         max(
           min(
-            DateTime.diff(step.inserted_at, recording_started_at, :millisecond) + 300,
+            DateTime.diff(step.inserted_at, recording_started_at, :millisecond) - 1000,
```

Comment:
In a follow-up, I'd change the timestamps to be a bit to be more tightly controlled by the `runner` – currently, we rely too much on the network and the step being inserted, which can move it on the timeline.

Right now, the steps are too far from the actual action happening, but we should remove this in the future and use more deterministic timestamps coming from the client.